COMMITTING.md

```diff
@@ -45,15 +45,14 @@ In general, most commits do not have a prefix.
 
 ### Example
 
-From [whatwg/html@54585](https://github.com/whatwg/html/commit/5458513792ab00d58e6c91ba48faaa611d034a2e):
-
 ```
-Editorial: move base URL from "module script" to "script"
+Editorial: Fix formatting and typo
+
+Fixes #35, fixes #38, fixes #21
+
+This is descriptive text providing details further explaining the title above. There is no restriction to the line length of this commit summary.
 
-This propagates an appropriate base URL for all instances of creating a
-classic script. This has no effect by itself (and indeed looks kind of
-pointless as of this commit), but it sets the stage for making import()
-use this base URL (#2315).
+Do remember to double space for subsequent paragraphs within summary.
```

Comment:
Wait, what? I don't think we should suggest double space.

Reply:
@zcorpan are you saying a commit summary of
_The quick brown fox jumped over the lazy dog.

Also the quick blue fox jumped over the lazy dog.
_

Should be written as
_The quick brown fox jumped over the lazy dog.
Also the quick blue fox jumped over the lazy dog.
_

This only came about because the commit message we used last year as an example had a double space between paragraphs. Am I wrong to think there should be a space in between? I CAN see how _"please add a space between paragraphs."_ OR just leave out the statement all together and ass-u-me the author knows how to write paragraphs. What say you? And will make the change ASAP. 🙏

Reply:
Ooh. I was confused, I was thinking about double-space between sentences.

Maybe say leave a blank line between paragraphs, instead?